tracer/test/Datadog.Trace.Tests/Agent/StatsAggregatorTests.cs

```diff
@@ -58,7 +58,7 @@ public async Task CallFlushAutomatically()
                     // Flush is not called if no spans are processed
                     aggregator.Add(new Span(new SpanContext(1, 1), DateTime.UtcNow));
 
-                    if (mutex.Wait(TimeSpan.FromMilliseconds(100)))
+                    if (mutex.WaitOrDump(TimeSpan.FromMilliseconds(100)))
```

Comment:
I'm worried about this one, 100ms is not that much considering that this is running in CI (all those VMs). Maybe we can bump this one a little bit more?

Reply:
I agree with you, though this one _isn't_ one we've seen fail IIRC, so I'm somewhat inclined to leave it for this PR (and bump it later)?